src/structures/GuildChannel.js

```diff
@@ -275,18 +275,23 @@ class GuildChannel extends Channel {
    *   .then(c => console.log(`Edited channel ${c}`))
    *   .catch(console.error);
    */
-  edit(data, reason) {
+  edit(editData, reason) {
+    let data = {
+      name: (editData.name || this.name).trim(),
+      topic: editData.topic,
+      position: this.rawPosition,
```

Comment:
go back to the previous code, but instead `position: typeof data.position === 'number' ? data.position : this.rawPosition,`

Reply:
Can't believe I didn't think of that.. Thanks! Will revert rn